EntityFramework.Exceptions.Common/ExceptionProcessorInterceptor.cs

```diff
@@ -107,6 +124,8 @@ private void SetConstraintDetails(DbContext context, UniqueConstraintException e
             exception.SchemaQualifiedTableName = match.SchemaQualifiedTableName;
         }
     }
+    
+    private record UniqueIndexDetail(string Name, string SchemaQualifiedName, IReadOnlyList<IProperty> Properties);
```

Comment:
There is already `IndexDetails` with the same properties, why not use it?

Reply:
Fixed 🤦